compiler-plugin/src/main/java/io/ballerina/stdlib/websub/Constants.java

```diff
@@ -55,4 +55,15 @@ public interface Constants {
     String BIN_DIR_NAME = "bin";
     String RESOURCES_DIR_NAME = "resources";
     String RESOURCE_DIR_NAME = "resource";
+
+    // code action related constants
+    String SERVICE_CONFIG_ANNOTATTION = "SubscriberServiceConfig";
+    String CONTENT_DISTRIBUTION_MESSAGE_TYPE = "ContentDistributionMessage";
+    String CONTENT_DISTRIBUTION_MESSAGE_PARAM_NAME = "message";
+    String ACKNOWLEDGEMENT_TYPE = "Acknowledgement";
+    String SUBSCRIPTION_DELETED_ERROR_TYPE = "SubscriptionDeletedError";
+    String BALLERINA_ERROR_PACKAGE = "annotations";
```

Comment:
Is this correct? Are you referring to `BALLERINA_ANNOTAIONS` or the Ballerina lang's generic `error` type?

Reply:
Yes, the package for `error` is `annotation` when we analyze the syntax-tree.